src/tests/Microsoft.PowerFx.Core.Tests/LazyRecordType.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Text;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class LazyRecordType : RecordType
```

Comment:
>LazyRecordType [](http://example.com/codeflow?start=17&length=14)

Can we just move this class next to the test in ValidateDisplayNames where it's used? (and make it a nested class) 
The thing is that it needs to stay totally in sync with that test, so having it as a nested class in that file will help with that.

Reply:
Updated.